### ISSUE TITLE
Make enable_fusing_conv2d_with_multiply_pattern default True

### ISSUE THREAD

**Issue by dgolubovicTT** [tenstorrent/tt-mlir#5258]
BatchNorm can be decomposed to add and matmul and fused to conv2d weight.
Afaik, we use this flag `enable_fusing_conv2d_with_multiply_pattern` to enable this.

@sdjordjevicTT can you tell why this is not True by default?

I tried to run resnet-50 on bfp8 and I get error: 

`BatchNorm only supports float32/bfloat16, got bfp8 (unsupported dtype)`

When I turn on this flag:`enable_fusing_conv2d_with_multiply_pattern` batch norm is decomposed and fused to conv2d weight and I don't get the assert.

@odjuricicTT what do you think about making this default?

**Comment by mtopalovicTT** [comment]
It's not on by default because there were models in forge which were failing on OOM because of this rewriter. Since I tested this at the time I have list of models which were failing on OOM and I tested a couple of them today and they seem to be passing now. I scheduled forge nightly https://github.com/tenstorrent/tt-forge-fe/actions/runs/18417041880 with this setting on by default to see if there is some change from last run.

One issue with which caused OOM was that after commuting multiply through conv2d what ends up being is that we multiply conv2d weight by scale where scale is in format (1, output_channels, 1, 1) and weight is in IOHW. Since both scale and weight are in tile when we multiply we bcast scale and after multiply we end up with weight, scale and multiply_result in DRAM all with IOHW shape (in tile layout) and we OOM.

We tried one fix by rewriting const_eval graph into RM but that had issues since ttnn ops don't support RM fully and in addition to that some ops which support RM just convert to Tile under the hood...

I think @sdjordjevicTT tried to execute const eval graphs in lingalg but I'm not sure how far that came along. I have one idea how to address this, but I would first wait for nightly to see how are things now.

**Comment by sdjordjevicTT** [comment]
Yeah, I have a branch where I managed to hoist const eval function to execute on CPU, and that actually solved problems. Unfortunately I didn't have time to continue working on it, but I will find someone to continue working on it soon.

**Comment by dgolubovicTT** [comment]
Thanks for the update. Yeah, cpu consteval is a must here...

@mtopalovicTT can you update this issue when you go through nightly if you find some new insights...
I see that there are a few OOM issues, few PCC issues and few models that just crash without the message...

**Comment by mtopalovicTT** [comment]
Yea it OOM so we can't just turn it on...
